keyvault/client/keys/src/main/java/com/azure/keyvault/keys/KeyAsyncClient.java

```diff
@@ -0,0 +1,594 @@
+     *
+     * <p><strong>Code Samples</strong></p>
+     * <p>Gets a specific version of the key in the key vault. Subscribes to the call asynchronously and prints out the
+     * returned key details when the callback happens.</p>
+     * <pre>
+     * String keyVersion = "6A385B124DEF4096AF1361A85B16C204";
+     * keyAsyncClient.getKey("keyName", keyVersion).subscribe(keyResponse -&gt;
+     *   System.out.printf("Key returned with name %s, id %s and version %s", keyResponse.value().name(),
+     *   keyResponse.value().keyId(), keyResponse.value().version()));
+     * </pre>
+     *
+     * @param name The name of the key, cannot be null
+     * @param version The version of the key to retrieve. If this is an empty String or null, this call is equivalent to calling {@link KeyAsyncClient#getKey(String)}, with the latest version being retrieved.
+     * @throws ResourceNotFoundException when a key with {@code name} and {@code version} doesn't exist in the key vault.
+     * @throws HttpRequestException if {@code name} or {@code version} are empty strings.
+     * @return A {@link Mono} containing a {@link Response} whose {@link Response#value() value} contains the requested {@link Key key}.
+     */
+    public Mono<Response<Key>> getKey(String name, String version) {
+        String keyVersion = "";
+        if(version != null){
+            keyVersion = version;
+        }
+        return service.getKey(endpoint, name, keyVersion, API_VERSION, ACCEPT_LANGUAGE, CONTENT_TYPE_HEADER_VALUE);
+    }
+
+    /**
+     * Get the public part of the latest version of the specified key from the key vault. The get key operation is applicable to all key types and
+     * If the requested key is symmetric, then no key material is released in the response. This operation requires the {@code keys/get} permission.
+     *
+     * <p><strong>Code Samples</strong></p>
+     * <p>Gets latest version of the key in the key vault. Subscribes to the call asynchronously and prints out the
+     * returned key details when the callback happens.</p>
+     * <pre>
+     * keyAsyncClient.getKey("keyName").subscribe(keyResponse -&gt;
+     *   System.out.printf("Key with name %s, id %s \n", keyResponse.value().name(),
+     *   keyResponse.value().keyId()));
+     * </pre>
+     *
+     * @param name The name of the key.
+     * @throws ResourceNotFoundException when a key with {@code name} doesn't exist in the key vault.
+     * @throws HttpRequestException if {@code name} is empty string.
+     * @return A {@link Mono} containing a {@link Response} whose {@link Response#value() value} contains the requested {@link Key key}.
+     */
+    public Mono<Response<Key>> getKey(String name) {
+        return getKey(name, "");
+    }
+
+
+    /**
+     * Get public part of the key which represents {@link KeyBase keyBase} from the key vault. If {@link KeyBase#version() version} is not set
```

Comment:
For this specific method, there is no way to set the version. Consider removing the description form the helper

Reply:
Updated the description.